bin/strata-cli/src/seed.rs

```diff
@@ -270,18 +270,35 @@ mod test {
     use super::*;
 
     #[test]
-    // Sanity check on private key construction
-    fn invalid_keys() {
-        // The key can't be zero
+    // Sanity checks on curve scalar construction, to ensure proper rejection
+    // This treats zero as invalid (for ECDSA reasons)
+    fn scalar_sanity_checks() {
+        // This is the (big-endian) order of the `secp256k1` curve group
```

Comment:
Can we have a reference link here somewhere?
Maybe this https://www.secg.org/sec2-v2.pdf with some guidance like "Section X.Y.Z"

Reply:
Good point! It's only a test, but does violate my personal "avoid undocumented magic numbers" rule...

Will update.

Reply:
It might even exist as a constant in one of the libraries that we could use, but I'm not really interested in tracking that down.

Reply:
Updated.